core/src/scss/utilities/mixins/link/_link-glyph.scss

```diff
@@ -0,0 +1,30 @@
+@charset "UTF-8";
+
+//
+// @link-glyph($glyph, $animate: true)
+//
+// Display a glyph after a link.
+//
+// $glyph {string} - Character to display after the link, e.g. '›', '\00BB'.
+// $animate {boolean} - Whether or not to animate on hover.
+//
+// Style guide: Mixins.Link.link-glyph
+//
+@mixin link-glyph($glyph, $animate: true) {
+  text-decoration: none;
+
+  &::after {
+    content: '#{$glyph}';
+    @include margin(null null -1px 0.4em);
+    transition: margin-left 0.2s ease-in-out;
+  }
+
+  &:hover,
+  &:focus {
+    &::after {
+      @if $animate {
+        @include margin(null null null 0.55em);
```

Comment:
instead of using margin to do the animation, try using `translateX()` as it does not affect the space around it and it will not cause reflow or shifting of other elements.

Reply:
Good tip. Thanks! This is done now.